lab_1/main.py

```diff
@@ -65,10 +82,19 @@ def get_top_n_words(freq_dict: dict, top_n: int) -> list:
     top_n = 1
     --> ['happy']
     """
-    pass
+    top_n_words = []
+    if isinstance(freq_dict, dict) and isinstance(top_n, int):
+        freq_dict = list(freq_dict.items())
+        freq_dict.sort(reverse=True)
```

Comment:
try this: freq_dict = {'a': 100, 'z': 2}
and this: freq_dict = {'a': 2, 'z': 100}

the result should be different